addons/zylann.hterrain/tools/importer/importer_dialog.gd

```diff
@@ -37,7 +40,13 @@ func _ready():
 		"raw_endianess": {
 			"type": TYPE_INT,
 			"usage": "enum",
-			"enum_items": ["Little Endian", "Big Endian"],
+			"enum_items": [[RAW_LITTLE_ENDIAN, "Little Endian"], [RAW_BIG_ENDIAN, "Big Endian"]],
+			"enabled": false
+		},
+		"bit_depth": {
+			"type": TYPE_INT,
+			"usage": "enum",
+			"enum_items": [[HTerrainData.BIT_DEPTH_UNDEFINED, "Undefined"], [HTerrainData.BIT_DEPTH_16, "16-bit"], [HTerrainData.BIT_DEPTH_24, "24-bit"]],
```

Comment:
> Though I guess it's also because that dropdown is also shown on non-raw files

Yes, that was my reason.

I wrote error messages in case the user has none defined for their .raw file. Or their chosen bit depth value is (probably) incorrect

Reply:
By default, the importer attempts to guess the bit depth. Technically, it's only undefined for non-raw files and if the user explicitly sets it as undefined.  
In the latter case, the import will throw a good error message in their face.

One possibility I could think of for improving the UX and preventing nonsense choices: **When enabling the OptionButton, remove that item**, and re-add it + select it when disabling the OptionButton

Reply:
OK… I used the [void set_item_disabled](https://docs.godotengine.org/en/latest/classes/class_optionbutton.html#class-optionbutton-method-set-item-disabled) option, because the bit depth identification check could fail…